### DISCUSSION TITLE
JavaScript Code Samples

### DISCUSSION THREAD

**Discussion by intelliot** [XRPLBounties/Proposals#31] [Open Bounties]
### Discussion thread for bounty: [JavaScript Code Samples](https://github.com/XRPLBounties/Proposals/blob/0a923c17e5cddcc7df317aade3ec678d5236bd2b/closed_bounties/0031%20JavaScript%20Code%20Samples.md) ([PR](https://github.com/XRPLBounties/Proposals/pull/3))

This bounty is currently open. Discuss below!

**Comment by intelliot** [comment]
Submission by @TusharPardhe here: https://github.com/XRPLF/xrpl-dev-portal/pull/1496

**Reply by free2ride19** [reply]
Open source' bounty approval @free2ride19

**Comment by jonathanlei** [comment]
Milestone 1 submissions:
by @roodtim here: https://github.com/XRPLF/xrpl-dev-portal/pull/1498
by @AlexanderBuzz here approved: https://github.com/XRPLF/xrpl-dev-portal/pull/1507

**Comment by JST5000** [comment]
We’ve accepted both @AlexanderBuzz’s and @TusharPardhe's submissions for milestone 1 for this bounty. Both will be paid out in full, and at this point we are no longer considering other submissions for Milestone 1. (The merged code is here: https://github.com/XRPLF/xrpl-dev-portal/pull/1507)

**Reply by JST5000** [reply]
If you have a confirmation code to verify your GitHub please add it in reply to this thread.

**Reply by TusharPardhe** [reply]
@JST5000 When you say confirmation code, what do we have to add?

**Reply by JST5000** [reply]
We added a new step to the process to email a 4 digit code to the email in the submitted bounty in order to verify they own the GitHub account. (Just emailed you)

Basically it's to avoid someone trying to claim someone else's work since GitHub is psuedo-anonymous.

**Reply by TusharPardhe** [reply]
Got it! Code: 1307

**Reply by AlexanderBuzz** [reply]
Hi Jackson, the Code is: 1736

**Reply by JST5000** [reply]
Thanks!

**Comment by AlexanderBuzz** [comment]
As nobody so far has claimed Milestone #2, I am going to take a shot at this one, should be a nice "between the years" project. Just you know someone will be taking care of that poor orphaned item.

**Comment by JST5000** [comment]
We've accepted @AlexanderBuzz's contribution for milestone 2!
* https://github.com/XRPLF/xrpl-dev-portal/pull/1673

Please submit this form to claim the bounty if you haven't already :) - [https://xrpl.typeform.com/xrpl-bounties](https://xrpl.typeform.com/xrpl-bounties?utm_source=xxxxx&utm_medium=xxxxx&utm_campaign=xxxxx&utm_term=xxxxx&utm_content=xxxxx)

**Comment by JST5000** [comment]
And with that, this bounty is completed! 🥂
